app/models/fuel/post.rb

```diff
@@ -11,7 +11,7 @@ class Post < ActiveRecord::Base
       attr_accessible :tag, :author_id, :content, :title, :teaser, :featured_image, :seo_title, :seo_description, :published_at
     end
 
-    has_attached_file :featured_image, :styles => { :medium => Fuel.configuration.featured_image_settings[:styles][:medium], :thumb => Fuel.configuration.featured_image_settings[:styles][:thumb] }, :default_url => "fuel/default-img.jpg"
+    has_attached_file :featured_image, :styles => { :medium => Fuel.configuration.featured_image_settings[:styles][:medium], :thumb => Fuel.configuration.featured_image_settings[:styles][:thumb] }, :default_url => "fuel/default-img.jpg", :storage => :s3, :s3_credentials => Proc.new{|a| a.instance.s3_credentials }
```

Comment:
Line is too long. [314/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space missing to the left of {.<br>Space between { and | missing.